controllers/controller.go

```diff
@@ -609,7 +613,10 @@ func (r *RpaasInstanceReconciler) cleanUpKEDAScaledObject(ctx context.Context, i
 }
 
 func isKEDAHandlingHPA(instance *v1alpha1.RpaasInstance) bool {
-	return instance.Spec.Autoscale != nil && instance.Spec.Autoscale.TargetRequestsPerSecond != nil && instance.Spec.Autoscale.KEDAOptions != nil && instance.Spec.Autoscale.KEDAOptions.Enabled
+	return instance.Spec.Autoscale != nil &&
+		(instance.Spec.Autoscale.TargetRequestsPerSecond != nil || len(instance.Spec.Autoscale.Schedules) > 0) &&
```

Comment:
I know we're just focusing on RPS right now, but wouldn't it be cleaner (since we're going to have other autoscaling parameters in the future) to just check if Keda is handling the autoscaling? What I mean is, should we really check if TargetRequestPerSecond is set here?

Reply:
Actually this PR is about scheduled window, not RPS.

> What I mean is, should we really check if TargetRequestPerSecond is set here?
We don't want to create ScaledObject when autoscaling targets are based on resource (CPU and memory).